### PR TITLE
Navbar for authenticated user; closes #125

### LINKED ISSUE #125: As authenticated user, I see a logout button and profile button in the nav bar, but not the login/signup

### PR DESCRIPTION
**Pull request by AdamSmith910**
Created a User show page with a corresponding controller action that displays a users information.

Created a 2 new integration tests in user_login_test.rb to ensure that an authenticated user was seeing a (working) "Profile" link, and not seeing the Login or Signup option in the navbar.

### CODE REVIEW

File: test/integration/user_login_test.rb [hunk 1]
```diff
@@ -35,6 +35,37 @@ def setup
...
+    refute page.has_content?("Login")
+  end
+
+  test "an authenticated user can access their profile" do
+    authenticated_user = create(:user)
+    ApplicationController.any_instance.stubs(:current_user).
+      returns(authenticated_user)
+
+    visit root_path
+    click_link_or_button "Profile"
```
Comment (EmilyMB) [inline test/integration/user_login_test.rb]: Yep! That's just like we discussed :smile:

File: app/views/layouts/_nav_bar.html.erb [hunk 2]
```diff
@@ -27,6 +27,7 @@
     <% else %>
       <%= menu_item "Welcome, #{current_user.username}" %>
```
Comment (EmilyMB) [inline app/views/layouts/_nav_bar.html.erb]: I think we discussed deleting this!

**Comment by EmilyMB** [comment]
Hey @AdamSmith910 This looks great. I would just delete the welcome message from the nav bar, and then this will be all set.

**Comment by jmejia** [comment]
:+1:

**Pull request merged**